sphinx/templates/latex/content.tex_t

```diff
@@ -1,12 +1,23 @@
 %% Generated by Sphinx.
 \def\sphinxdocclass{<%= docclass %>}
+<% if latex_engine == 'lualatex' -%>
+\IfFileExists{luatex85.sty}
+ {\RequirePackage{luatex85}}
+ {\ifdefined\luatexversion\ifnum\luatexversion>84\relax
+  \PackageError{sphinx}
+  {** With this LuaTeX (\the\luatexversion),Sphinx requires luatex85.sty **}
+  {** Add the LaTeX package luatex85 to your TeX installation, and try again **}
+  \endinput\fi\fi}
+<% endif -%>
 \documentclass[<%= papersize %>,<%= pointsize %><%= classoptions %>]{<%= wrapperclass %>}
 \ifdefined\pdfpxdimen
    \let\sphinxpxdimen\pdfpxdimen\else\newdimen\sphinxpxdimen
 \fi \sphinxpxdimen=<%= pxunit %>\relax
 <%= passoptionstopackages %>
 <%= geometry %>
+<% if latex_engine == 'pdflatex' -%>
```

Comment:
memo: about `utf8extra`,  in (distant) future, it could use http://www.ctan.org/pkg/uspace (https://github.com/wilx/project-uspace) which gives meaning to Unicode space characters. It is small package but with a dependency on [newunicodechar](http://www.ctan.org/pkg/newunicodechar) latex package. Actually the latter is not big either. We could possibly ship them with sphinx.